### PR TITLE
Action Menu protocol

### PR DESCRIPTION
**Pull request by rodolfomiranda**
Adding the Action Menu protocol 1.0 and 2.0. 
2.0 is the same as 1.0 with header translated to body. Please, confirm if it should still be version 2.0.

### CODE REVIEW

File: site/content/protocols/action-menu/2.0/readme.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+---
+title: Mediator Coordinator
```
Comment (genaris) [inline site/content/protocols/action-menu/2.0/readme.md:2]: Title should be Action Menu

File: site/content/protocols/action-menu/2.0/readme.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+--- | ---
+                     |       
+                     |       
+
+
+
+## Message Reference
+
+### Menu
+A `requestor` is expected to display only one active menu per connection when action menus are employed by the `responder`. A newly received menu is not expected to interrupt a user, but rather be made available for the user to inspect possible actions related to the `responder`.
```
Comment (genaris) [inline site/content/protocols/action-menu/2.0/readme.md:77]: Roles are defined as `requester` and  `responder`. In 1.0 protocol, requester role wording was a bit mixed up and seems to be fixed here, but this `requestor` is still there.

**Review by genaris — COMMENTED** [review]
I left a few minimal typos or wording comments.

In this version bump it is possible to introduce other modifications to the protocols than adding the body and DIDComm v2-related stuff? Or they should be addressed in Aries RFCs and then adopted here?

File: site/content/protocols/action-menu/2.0/readme.md (hunk 2 above)
Comment (rodolfomiranda) [reply site/content/protocols/action-menu/2.0/readme.md]: fixed

**Comment by rodolfomiranda** [comment]
> I left a few minimal typos or wording comments.
> 
> In this version bump it is possible to introduce other modifications to the protocols than adding the body and DIDComm v2-related stuff? Or they should be addressed in Aries RFCs and then adopted here?

Yes, it is posible to add changes. I understand that from now on, following protocol versions will be in sync with Aries RFC to supporting both type of message formats (if applicable)

**Pull request merged**